src/main/java/com/shaft/gui/image/ScreenshotManager.java

```diff
@@ -778,4 +778,7 @@ private static synchronized void appendToAnimatedGif(byte[] screenshot) {
             ReportManagerHelper.log(e);
         }
     }
+    public static String getGifRelativePathWithFileName() {
```

Comment:
Can we make the gif method return its oath instead?

Reply:
Done. I didn't before because _attachAnimatedGif()_ was clearing the path at the end.